Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ######## Update SGX SDK ########
-include UpdateRustSGXSDK.mk
+#include UpdateRustSGXSDK.mk
```

Comment:
Do not update the SDK on each build. The last SDK update had breaking changes which prevented us from building the worker in a 'clean' environment

Reply:
The same "comment" is no line 30. I think we should delete one of the two and add more explaining (e.g. what you write here) as comment.

Reply:
agreed, I'll clean it up 👍